server/app/tests/test.js

```diff
@@ -0,0 +1,183 @@
+import expect from 'expect';
+import request from 'supertest';
+
+import app from './../../app';
+
+import {
+  Orders,
+  menu,
+} from '../db/dbconnect';
+
+
+describe('GET /api/v1/menu', () => {
+  it('should return menu of avaliable food', (done) => {
+    request(app)
+      .get('/api/v1/menu')
+      .set('Accept', 'application/json')
+      .expect(200)
+      .expect((res) => {
+        expect(res.body.data === menu);
+        expect(200);
+      })
+      .end(done);
+  });
+});
+
+describe('GET /api/v1/orders', () => {
+  it('should return a 404 no order has been made', (done) => {
+    request(app)
+      .get('/api/v1/orders')
+      .expect(404)
+      .end(done);
+  });
+
+  it('respond with json containing a list of all orders', (done) => {
+    const data = {
+      foodId: 2,
+      quantity: 7,
+    };
+    Orders.push(data);
+    request(app)
+      .get('/api/v1/orders')
+      .set('Accept', 'application/json')
+      .expect(200)
+      .expect((res) => {
+        expect(res.body.data === Orders);
+        expect(200);
+      })
+      .end(done);
+  });
+});
+
+
+describe('POST /api/v1/orders', () => {
+  const data = {
+    foodId: 1,
+    quantity: 6,
+  };
+
+  it('it should post order successfully', (done) => {
+    request(app)
+      .post('/api/v1/orders')
+      .send(data)
+      .expect(200)
+      .expect((res) => {
+        expect(res.body.status === 'success');
+        expect(200);
+      })
+      .end(done);
+  });
+
+  it('it should return a 404 "not found"', (done) => {
+    const item = {
+      foodId: 700,
+      quantity: 7,
+    };
+    request(app)
+      .post('/api/v1/orders')
+      .send(item)
+      .set('Accept', 'application/json')
+      .expect(404)
+      .end(done);
+  });
+
+  it('it should not place an order when data is empty', (done) => {
```

Comment:
This works really well as test-case description, please use this method in all descriptions.

Reply:
done